app/Http/Controllers/IdeasController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\StoreIdea;
+use App\Idea;
+
+class IdeasController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+    
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index($id)
+    {
+ 
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(StoreIdea $request)
+    {
+        $validated = $request->validated();
```

Comment:
I don't see `$validated` being used anywhere. Same as the `BoardsController`

Reply:
It's still validated as per the Laravel Documentation -
"You may type-hint any dependencies you need within the rules method's signature. They will automatically be resolved via the Laravel service container.

So, how are the validation rules evaluated? All you need to do is type-hint the request on your controller method. The incoming form request is validated before the controller method is called, meaning you do not need to clutter your controller with any validation logic. "

https://laravel.com/docs/5.7/validation#creating-form-requests

Reply:
That means `$request->validated()` is not needed if you are using form requests. When you are not using form requests, you would use `$request->validated()`. Those are 2 different options.

Reply:
Much clearer now, thanks!